src/siphon/metadata.py

```diff
@@ -513,7 +511,7 @@ def _get_handler(self, handler_name):
         elif handler_name in self._sts:
             return getattr(self._st, handler_name)
         else:
-            msg = 'cannot find handler for element {}'.format(handler_name)
+            msg = f'cannot find handler for element {handler_name}'
             log.warning(msg)
```

Comment:
Want to go ahead and flake8-pieify this as a log message (didn't dig for more outside of changes) while you're in here?

Reply:
Sure. Did yours catch this? Because mine didn't.

Reply:
I didn't run it myself, I just remembered making that change over on MetPy. I can check and see if it does.

Reply:
Update: it doesn't.